src/components/Reviews/index.js

```diff
@@ -17,17 +20,34 @@ class Reviews extends Component {
   render() {
     const { classes, reviews, loading } = this.props
 
-    if (loading) {
-      return 'Loading'
-    } else {
-      return <Grid container spacing={16} className={classes.root}>
-        {reviews.map((review, index) => {
-          return <Grid key={index} item xs={12} > 
-            <Review review={review} key={reviews.length + index}/>
-          </Grid>
-        })}
-      </Grid>
-    }
+    return <Grid container spacing={16} className={classes.root}>
+      {loading && <Grid item xs={12}>
+        <Card>
+          <CardHeader
+            avatar={<Avatar><LoadingIcon/></Avatar>}
+            title='Fetching reviews...'
```

Comment:
Should we use a circular spinner here?

Reply:
Good point, we should also put it in the other places where I used this kind of loading state display (transactions view)